devito/operator.py

```diff
@@ -186,6 +186,22 @@ def signature(self):
         return self.input_params + [param for param in self.output_params
                                     if param not in self.input_params]
 
+    @property
+    def total_kernel_time(self):
+        return self.propagator.timings[0]
+
+    @property
+    def total_space_loops_time(self):
+        return self.propagator.timings[1]
+
+    @property
+    def total_pre_loops_time(self):
+        return self.propagator.timings[2]
+
+    @property
+    def total_post_loops_time(self):
```

Comment:
To be honest I have not seen those names anywhere in the code. I used them just because they were much shorter than `time_loops_stencils`. 

Right now I am working on completely redoing the profiler by making a profiler class. I will close this PR until the new version is ready (it should be ready tomorrow) if it's not urgent to have it merged.